typescript/cli/src/utils/keys.ts

```diff
@@ -77,14 +78,14 @@ async function addressToImpersonatedSigner(
  * @param key a private key
  * @returns a signer for the private key
  */
-function privateKeyToSigner(key: string): ethers.Wallet {
+function privateKeyToSigner(key: string): Wallet {
   if (!key) throw new Error('No private key provided');
 
   const formattedKey = key.trim().toLowerCase();
   if (ethers.utils.isHexString(ensure0x(formattedKey)))
-    return new ethers.Wallet(ensure0x(formattedKey));
+    return new Wallet(ensure0x(formattedKey));
```

Comment:
It has to be backwards compatible because its used in bridging transactions from L2 to L1.
+ So far, we have had no issues using zksync wallet to deploy/send transactions on EVM.